pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -120,14 +123,23 @@ func New(cidr *net.IPNet, allocatorFactory allocator.AllocatorFactory) (*Range,
 	}
 	var err error
 	r.alloc, err = allocatorFactory(r.max, rangeSpec)
-
-	return &r, err
+	if err != nil {
+		return nil, err
+	}
+	// default $min = 16, $max = 256 and $step = 16.
+	offset := allocator.DynamicRangeOffset(16, 256, 16, int(max))
```

Comment:
we only need to export this function to be able to set the following metric

Reply:
See my other comments - I am not sure about this metric...